src/py_eddy_tracker/observations/network.py

```diff
@@ -97,6 +101,109 @@ def fix_next_previous_obs(next_obs, previous_obs, flag_virtual):
         previous_obs[i_o + 1] = i_o
 
 
+def advect(x, y, c, t0, delta_t):
```

Comment:
Maybe update exemple with this new method/function.
I think those methods must be callable in classic tracking also

Reply:
function moved to groups, and example updated